### ISSUE TITLE
bug: Threads stop working randomly, using Anthropic API

### ISSUE THREAD

**Issue by madacol** [janhq/jan#3002]
**Describe the bug**
I just installed Jan and started testing the Claude models, and after editing a message in my first thread and submitting it, the UI showed an error of "Overloaded", then I tried editing in another thread, and it also failed but with a more generic error (can't remember which).
All subsequent retries in any of those broken threads now fail with the error: "Invalid API key", which is wrong because I can still use the same model on other threads

**Steps to reproduce**
I haven't been able to break another thread, but I still have those  broken threads that if I try to edit or add a new message, they fail with "Invalid API key".
And both threads in their `messages.jsonl` file have as the last message an assistant response that says: `messages: roles must alternate between \"user\" and \"assistant\", but found multiple \"user\" roles in a row`

**Environment details**
- Operating System: Ubuntu
- Jan Version: 0.5.0

**Additional context**
messages.jsonl of a broken thread
```jsonl
{"id":"01HZPKMAT2GANGVWPM4W78SW41","thread_id":"jan_1717666232","role":"user","status":"ready","created":1717671963458,"updated":1717671963458,"object":"thread.message","content":[{"type":"text","text":{"value":"test","annotations":[]}}]}
{"id":"01HZPKMATS0Q4401RKX5GTNAHD","thread_id":"jan_1717666232","type":"Thread","role":"assistant","content":[{"type":"text","text":{"value":"messages: roles must alternate between \"user\" and \"assistant\", but found multiple \"user\" roles in a row","annotations":[]}}],"status":"error","created":1717671963481,"updated":1717671963481,"object":"thread.message","error_code":"invalid_request_error"}
```

**Comment by madacol** [comment]
It happened again today without editing, just sending a normal message

![image](https://github.com/janhq/jan/assets/6379896/fbfa9817-1d29-4648-b63a-29423a85e966)

after retrying

![image](https://github.com/janhq/jan/assets/6379896/cb53c938-5d27-4f79-b202-30dd178d4e06)

**Comment by madacol** [comment]
I just tested removing the last 2 lines from `messages.jsonl` and the thread started working again

**Comment by Van-QA** [comment]
The issue is resolved in Jan nightly Jan v0.5.0-473, feel free to try the nightly, or st‌ay tuned for the sta‌ble release :‌pray:
